nsxt/resource_nsxt_policy_lb_https_monitor_profile.go

```diff
@@ -0,0 +1,194 @@
+/* Copyright © 2020 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/vsphere-automation-sdk-go/runtime/bindings"
+	"github.com/vmware/vsphere-automation-sdk-go/runtime/data"
+	"github.com/vmware/vsphere-automation-sdk-go/runtime/protocol/client"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/infra"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+)
+
+var lBServerSslProfileBindingServerAuthValues = []string{
+	model.LBServerSslProfileBinding_SERVER_AUTH_REQUIRED,
+	model.LBServerSslProfileBinding_SERVER_AUTH_IGNORE,
+	model.LBServerSslProfileBinding_SERVER_AUTH_AUTO_APPLY,
+}
+
+func resourceNsxtPolicyLBHttpsMonitorProfile() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceNsxtPolicyLBHttpsMonitorProfileCreate,
+		Read:   resourceNsxtPolicyLBHttpsMonitorProfileRead,
+		Update: resourceNsxtPolicyLBHttpsMonitorProfileUpdate,
+		Delete: resourceNsxtPolicyLBHttpsMonitorProfileDelete,
+		Importer: &schema.ResourceImporter{
+			State: nsxtPolicyPathResourceImporter,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"nsx_id":                     getNsxIDSchema(),
+			"path":                       getPathSchema(),
+			"display_name":               getDisplayNameSchema(),
+			"description":                getDescriptionSchema(),
+			"revision":                   getRevisionSchema(),
+			"tag":                        getTagsSchema(),
+			"request_body":               getLbMonitorRequestBodySchema(),
+			"request_header":             getLbHTTPHeaderSchema("Array of HTTP request headers"),
+			"request_method":             getLbMonitorRequestMethodSchema(),
+			"request_url":                getLbMonitorRequestURLSchema(),
+			"request_version":            getLbMonitorRequestVersionSchema(),
+			"response_body":              getLbMonitorResponseBodySchema(),
+			"response_status_codes":      getLbMonitorResponseStatusCodesSchema(),
+			"server_ssl_profile_binding": getLbServerSslProfileBindingSchema(),
```

Comment:
I suggest to call this `server_ssl` for consistency with other LB resources (f.e. virtual server)